MAUI/NumericEntry/Basic-Features.md

```diff
@@ -268,4 +268,110 @@ sfNumericEntry.VerticalTextAlignment = TextAlignment.Start;
 
 The following image illustrates the result of the above code:
 
-![HorizontalTextAlignment](GettingStarted_images/textalignment.png)
\ No newline at end of file
+![HorizontalTextAlignment](GettingStarted_images/textalignment.png)
+
+## Set the compute to percentage in SfNumericEntry
+
+When the [SfNumericEntry](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.SfNumericEntry.html) is in percentage format, the value can be displayed in two ways as follows:
+
+`Value`: Displays the actual value with percentage symbol.
+
+![.NET MAUI NumericEntry PercentDisplayMode](GettingStarted_images/percentdisplaymode_value.png)
```

Comment:
@SivaranjithN Add the output image after the code snippet

Reply:
Addressed the comments.